pyproject.toml

```diff
@@ -48,6 +48,7 @@ dev = [
   "mypy-boto3-s3==1.34.120",
   "mypy==1.10.0",
   "numpy==1.22.2",
+  "polars>=1.24.0",
```

Comment:
Will be more useful later on in further work when we want to do analysis of the stored benchmarks data. I'm interested to explore using polars in more places (maybe also in Kraken) and this seems a safe place to play with it (it's only added as a dev dependency).